public/mockServiceWorker.js

```diff
@@ -1,33 +1,67 @@
 /**
  * Mock Service Worker.
- * @see https://github.com/kettanaito/msw
- * This Service Worker is meant for development usage only.
- * Make sure not to include it on production.
+ * @see https://github.com/open-draft/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
  */
-self.addEventListener('install', (event) => {
+/* eslint-disable */
+/* tslint:disable */
+
+const INTEGRITY_CHECKSUM = '582e1e1f1f2b85a3997e0537abd5fe4d'
```

Comment:
This is the missing variable that failed the integrity check for you previously.